index.html

```diff
@@ -218,7 +245,16 @@ <h3>legend</h3>
         }
 
         // Add event listener for opening and closing details
-        $('#mozPositions').find('tbody').on('click', 'td', function () {
+        $('#mozPositions').find('tbody').on('click', 'td', function (event) {
+          var a = $(event.target).closest('a');
+          if (a.length > 0 && !a[0].parentNode.classList.contains("details-control")) {
+            // Do nothing if the click was on a link, except for the
+            // details-control.
```

Comment:
Thanks for fixing this! This had been a papercut that I kept meaning to look into.